sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationMockTests.cs

```diff
@@ -295,16 +295,75 @@ public async Task AnalyzeOperationWithActionsError()
             Assert.Throws<InvalidOperationException>(() => entitiesActionsResults.DocumentsResults.GetType());
 
             Assert.IsTrue(keyPhrasesActionsResults.HasError);
-            Assert.Throws<InvalidOperationException>(() => entitiesActionsResults.DocumentsResults.GetType());
+            Assert.Throws<InvalidOperationException>(() => keyPhrasesActionsResults.DocumentsResults.GetType());
 
             Assert.IsTrue(piiActionsResults.HasError);
-            Assert.Throws<InvalidOperationException>(() => entitiesActionsResults.DocumentsResults.GetType());
+            Assert.Throws<InvalidOperationException>(() => piiActionsResults.DocumentsResults.GetType());
 
             Assert.IsTrue(entityLinkingActionsResults.HasError);
-            Assert.Throws<InvalidOperationException>(() => entitiesActionsResults.DocumentsResults.GetType());
+            Assert.Throws<InvalidOperationException>(() => entityLinkingActionsResults.DocumentsResults.GetType());
 
             Assert.IsTrue(analyzeSentimentActionsResults.HasError);
-            Assert.Throws<InvalidOperationException>(() => entitiesActionsResults.DocumentsResults.GetType());
+            Assert.Throws<InvalidOperationException>(() => analyzeSentimentActionsResults.DocumentsResults.GetType());
+        }
+
+        [Test]
+        public void AnalyzeOperationWithGenericError()
```

Comment:
mocking a test just to be able to test what will happen and guarantee that .NET won't break without telling the customer why